pkgs/applications/virtualization/xhyve/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "xhyve-${version}";
+  version = "0.1.0";
+
+  src = fetchurl {
+    url = "https://github.com/mist64/xhyve/archive/v${version}.tar.gz";
+    sha256 = "0nbb9zy4iqmdz2dpyvcl1ynimrrpyd6f6cq8y2p78n1lmgqhrgkm";
+  };
+
+  buildPhase = ''
```

Comment:
Please use `buildFlags` instead of overwriting the `buildPhase`. See http://nixos.org/nixpkgs/manual/.